### DISCUSSION TITLE
Can time-based Tariffs be added to the data?

### DISCUSSION THREAD

**Discussion by azsoccerpop** [jertel/vuegraf#161] [Q&A]
This may be more of an influxdb question than one for the integration. My electric grid company has peak and off-peak pricing based on both time-of-day and month. Is there a way to tag the data as it is written with "peak" or "off-peak" based on a schedule?

**Comment by jertel** [comment]
It's possible but will require Python coding. I wonder though if it makes more sense to do that determination on the reporting end. In other words, determine if it was peak or not at the time of reading and displaying the data rather than at the time of writing the data.

**Comment by azsoccerpop** [comment]
admittedly I am very new to influxdb and grafana and I imagine it is possible from the reporting end. I will research this in Grafana based on the time series data. Thank you for your work on this as I am less reliant on cloud-based reporting which is a good thing.
